### ISSUE TITLE
Change how core responses are being parsed

### ISSUE THREAD

**Issue by rishabhpoddar** [supertokens/supertokens-golang#21]
- Avoid marshalling and then unmarshalling
- The inbuilt golang lib is slow. Maybe checkout https://github.com/mitchellh/mapstructure or https://github.com/json-iterator/go (working example: https://play.golang.org/p/dwZow8iLgJT)

**Comment by DevajMody** [comment]
@rishabhpoddar the x is MB/s. S, L and XL are the size of the payloads.

**Comment by rishabhpoddar** [comment]
@DevajMody true. I get it now. The API responses are largely small JSONs only. So km closing this. Thanks
